win32/build/mkdist.php

```diff
@@ -118,6 +118,8 @@ function get_depends($module)
 		}
 
 		$per_module_deps[basename($module)][] = $dep;
```

Comment:
A minor issue here. With the addition of recursion, `$dep` might be added multiple times. Please see file `snapshot.txt` after the packaging.

Thanks.